src/main/java/randoop/operation/TypedOperation.java

```diff
@@ -367,9 +368,11 @@ public static TypedClassOperation forConstructor(Constructor<?> constructor) {
    * Constructs a {@link TypedOperation} for a method object.
    *
    * @param method the reflective method object
+   * @param accessibilityPredicate AccessibilityPredicate to check if the method is accessible
```

Comment:
You don't need to repeat the parameter's type, which is apparent nearby, in both the code and the generated API documentation.